src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -118,7 +118,7 @@
 
 (defn- max-fee [gas gas-price]
   (when (and gas gas-price)
-    (money/wei->ether (.times gas gas-price))))
+    (money/wei->ether (.times (money/bignumber gas) gas-price))))
```

Comment:
Why this change? Isn't `gas` always a bignumber at this point?

Reply:
It wasn't. Took me a while to figure that.

Reply:
Not when it arrives from bot via signal anyway

Reply:
Can we make it a bignumber at the source then? All our numerical values should be bignumber, and wei when relevant, to reduce confusion potential.